mappings/net/minecraft/world/ChunkSerializer.mapping

```diff
@@ -11,23 +11,39 @@ CLASS net/minecraft/unmapped/C_bsqbntcq net/minecraft/world/ChunkSerializer
 	FIELD f_torykzsj SECTIONS_KEY Ljava/lang/String;
 	FIELD f_xylwjepg Z_POS_KEY Ljava/lang/String;
 	FIELD f_yfcoouba CODEC Lcom/mojang/serialization/Codec;
+	METHOD m_aeqdhxlr (Lnet/minecraft/unmapped/C_ynrszrtu;ILjava/lang/String;)V
+		ARG 2 error
+	METHOD m_afhfkuml (Lnet/minecraft/unmapped/C_hhlwcnih;Lnet/minecraft/unmapped/C_oivssbvb;)V
+		ARG 1 data
 	METHOD m_akmhowly appendTickNbt (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_hhlwcnih;Lnet/minecraft/unmapped/C_lwzmmmqr$C_kflzqqfb;)V
 		ARG 1 nbt
 		ARG 2 ticksToSave
 	METHOD m_bvwxfvxl writeStructures (Lnet/minecraft/unmapped/C_ywyzxrzo;Lnet/minecraft/unmapped/C_ynrszrtu;Ljava/util/Map;Ljava/util/Map;)Lnet/minecraft/unmapped/C_hhlwcnih;
 		ARG 0 context
 		ARG 1 pos
+		ARG 2 structureStarts
+		ARG 3 structureReferences
 	METHOD m_drlgdfvh (Lnet/minecraft/unmapped/C_rxhyurmy;)Ljava/lang/String;
 		ARG 0 fluid
+	METHOD m_dsxrwnsw (Ljava/lang/String;)Ljava/util/Optional;
+		ARG 0 fluidId
 	METHOD m_dusmokor (Lnet/minecraft/unmapped/C_hqhtfwlr;Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_hqhtfwlr;Lnet/minecraft/unmapped/C_hrdsvlkq;)V
 		ARG 3 chunk
+	METHOD m_evfcpwdf (Ljava/lang/String;)Ljava/util/Optional;
+		ARG 0 blockId
 	METHOD m_ezqkremw toNbt ([Lit/unimi/dsi/fastutil/shorts/ShortList;)Lnet/minecraft/unmapped/C_hqhtfwlr;
 		ARG 0 lists
 	METHOD m_gcmtnulo getChunkType (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_kogtzhzt$C_lcffkbeg;
 		ARG 0 nbt
+	METHOD m_iuzckbsr (Ljava/lang/String;)Ljava/util/Optional;
+		ARG 0 blockId
 	METHOD m_mxkjjsvz getList (Lnet/minecraft/unmapped/C_hhlwcnih;Ljava/lang/String;)Lnet/minecraft/unmapped/C_hqhtfwlr;
 		ARG 0 nbt
 		ARG 1 key
+	METHOD m_nxqmhxhy (Lnet/minecraft/unmapped/C_ynrszrtu;Lnet/minecraft/unmapped/C_ncpywfca;J)Z
+		ARG 2 l
```

Comment:
```suggestion
		ARG 2 pos
```
why is this one just `l`?

Reply:
you caught me being lazy :p
that's a lambda and I didn't verify if it was actually a position or not
will check in a bit!

Reply:
it is indeed a position, thanks!